staff/meetings/meetings.py

```diff
@@ -0,0 +1,175 @@
+from collections import defaultdict
+from datetime import datetime
+from getpass import getuser
+from math import ceil
+from os import listdir, makedirs
+from os.path import expanduser, exists, join
+from re import compile
+from shutil import copyfile
+from time import strftime
+
+
+def get_minutes_folder():
+    return expanduser("~staff/minutes")
+
+
+def ls(state=None):
+    states = get_membership_status(get_current_minutes_path("bod"))
+    if state:
+        states = {k: v for k, v in states.items() if v == state}
+    return(states)
+
+
+def quorum():
+    bod = ls(state="bod")
+    return(int(ceil(2 / 3 * len(bod))))
+
+
+def get_template(choice):
+    minutes_folder = get_minutes_folder()
+    if exists(join(minutes_folder, choice, "template")):
+        return join(minutes_folder, choice, "template")
+    return join(minutes_folder, "template")
+
+
+def get_minutes_choices():
+    return(listdir(get_minutes_folder()))
+
+
+def get_current_minutes_path(choice):
+    return get_minutes_path(choice, get_semester())
+
+
+def get_minutes_file():
+    return strftime("%Y-%m-%d")
+
+
+def get_minutes_path(choice, semester):
+    path = join(get_minutes_folder(), choice)
+    if not exists(path):
+        raise ValueError("argument must be from get_minutes_choices()")
+    path = join(path, semester)
+    if not exists(path):
+        makedirs(path)
+    return path
+
+
+def get_semester():
+    now = datetime.now()
+    year = now.year
+    month = now.month
+
+    AUG = 9
+    MAY = 5
+    # a reasonable approximation for semesters
+    if month > AUG or (month == AUG and day > 20):
+        sem = "Fall"
+    elif month < MAY or (month == MAY and day < 20):
+        sem = "Spring"
+    else:
+        sem = "Summer"
+    return(join(str(year), sem))
```

Comment:
we can almost use the logic in ocflib for this, i guess this is fine for now though